lib/elixir/lib/kernel/utils.ex

```diff
@@ -22,6 +22,8 @@ defmodule Kernel.Utils do
         _ -> raise ArgumentError, "invalid syntax in defdelegate #{Macro.to_string(fun)}"
       end
 
+    check_for_default_args(args)
```

Comment:
Well that makes sense. I was too focused on stuff that people may want to use here that I overlooked stuff that people *could* use :) Fixing in a couple hours